### PR TITLE
Cover some test/widgets tests with leak tracking

### PR DESCRIPTION
**Pull request by ksokolovskyi**
### Description
This is the fixed copy of the https://github.com/flutter/flutter/pull/133767.

This PR changes `testWidgets` by `testWidgetsWithLeakTracking` in number of `flutter/test/widgets` tests according to the https://github.com/dart-lang/leak_tracker/issues/134.

### Marks leaks
- https://github.com/flutter/flutter/issues/134205

### Tests
- Updates `absorb_pointer_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `actions_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `align_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_align_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_container_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_cross_fade_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_grid_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_image_filtered_repaint_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_list_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_opacity_repaint_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_padding_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_positioned_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_size_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `animated_switcher_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `annotated_region_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `app_lifecycle_listener_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `app_navigator_key_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `app_overrides_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `app_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `app_title_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `aspect_ratio_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `async_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `autocomplete_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `autofill_group_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `automatic_keep_alive_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `backdrop_filter_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `banner_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `baseline_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `basic_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `binding_deferred_first_frame_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `binding_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `box_decoration_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `build_context_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `build_scope_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `center_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `clamp_overscrolls_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `clip_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `color_filter_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `column_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `composited_transform_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `constrained_box_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `container_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `context_menu_controller_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `coordinates_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `custom_multi_child_layout_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `custom_paint_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `custom_painter_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `custom_scroll_view_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `custom_single_child_layout_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `debug_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `decorated_sliver_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `default_colors_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `default_text_editing_shortcuts_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `default_text_height_behavior_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `default_text_style_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `did_update_widget_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `directionality_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `dismissible_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `display_feature_sub_screen_test.dart` to use `testWidgetsWithLeakTracking`;
- Updates `disposable_build_context_test.dart` to use `testWidgetsWithLeakTracking`.

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] All existing and new tests are passing.

### CODE REVIEW

**Comment by ksokolovskyi** [comment]
@polina-c please review this PR when you have time.

File: packages/flutter/test/widgets/actions_test.dart [hunk 1]
```diff
@@ -392,9 +393,11 @@ void main() {
       await buildTest(true);
       expect(hovering, isFalse);
       expect(focusing, isFalse);
+
+      focusNode.dispose();
```
Comment (polina-c) [inline packages/flutter/test/widgets/actions_test.dart:397]: Can you update disposals to be aligned with newly gained knowledge: https://github.com/dart-lang/leak_tracker/pull/138

**Review by polina-c — CHANGES_REQUESTED** [review]
One small comment.

File: packages/flutter/test/widgets/actions_test.dart (hunk 1 above)
Comment (ksokolovskyi) [reply packages/flutter/test/widgets/actions_test.dart]: @polina-c great rule, thanks.

**Comment by ksokolovskyi** [comment]
@polina-c Could you please take a look again?

**Review by polina-c — APPROVED** [review]

**Comment by ksokolovskyi** [comment]
@polina-c could you please take a look again?

**Pull request merged**